docs/conf/portland/2025/cfp.rst

```diff
@@ -99,42 +91,44 @@ Past speaker roles include but are not limited to
 What we’re looking for
 ----------------------
 
-**Diverse topics**
+**Broad topics**
 
 The focus of Write the Docs is software documentation, and we actively seek talks that address a wide range of related subjects, at various levels of expertise.
 Documentation perspectives from other fields are welcome, as are topics from adjacent fields!
 
 **Practicality and positivity**
 
-We prefer talks backed by experience and experimentation to talks about theory, and we definitely don't like talks that bad-mouth technologies or approaches.
-Don't tell us why you hate something – tell us how you overcame the problems it was causing.
+We prefer talks backed by experience and experimentation to talks about theory, and we definitely don't like talks that bad-mouth technologies or approaches. Don't tell us why you hate something – tell us how you overcame the problems it was causing.
 
 **Process over tooling**
 
-We avoid talks about specific tooling, which often turn into marketing pitches or tutorials.
-We would much rather hear about process, culture, data, people, or the metaphysical side effects of spending your life thinking about docs.
+We tend to avoid talks about specific tools, which often turn into marketing pitches or tutorials. We would much rather hear about process, culture, data, people, or the metaphysical side effects of spending your life thinking about docs.
```

Comment:
```suggestion
We tend to avoid talks about specific tooling, which often turn into marketing pitches or tutorials. We would much rather hear about process, culture, data, people, or the metaphysical side effects of spending your life thinking about docs.
```